### PR TITLE
docs: Document Ansible logging configuration

### PR DESCRIPTION
**Pull request by NicolasT**
Fixes: #255
See: https://github.com/scality/metalk8s/issues/255
See: RFE-152

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# We have `ansible.cfg` in VCS, but because one may make local changes, we don't
```
Comment (Zempashi) [inline .gitignore:1]: Or part of a make shell, copy over an ansible.cfg.sample. Only the sample is checked in git and allow users to modify the ansible.cfg at will (or loose it if making another git clone)

**Review by Zempashi — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (NicolasT) [reply .gitignore:1]: Yeah, that's a sensible approach (though I wouldn't let `make shell` create a copy, that's a bit IMHO).

**Review by wabernat — DISMISSED** [review]
Approved for language, with edits.

**Comment by NicolasT** [comment]
Needs new branch name.

**Pull request closed without merging**